parts/windows/windowsconfigfunc.ps1

```diff
@@ -237,19 +237,10 @@ function Install-GmsaPlugin {
     }
     Remove-Item -Path $tempPluginZipFile -Force
 
-    $tempInstallPackageFoler = [Io.path]::Combine($tempInstallPackageFoler, "CCGPlugin")
-
     # Copy the plugin DLL file.
     Write-Log "Installing the GMSA plugin"
     Copy-Item -Force -Path "$tempInstallPackageFoler\CCGAKVPlugin.dll" -Destination "${env:SystemRoot}\System32\"
 
-    # Enable the logging manifest.
-    Write-Log "Importing the CCGEvents manifest file"
-    wevtutil.exe im "$tempInstallPackageFoler\CCGEvents.man"
-    if ($LASTEXITCODE) {
-        Set-ExitCode -ExitCode $global:WINDOWS_CSE_ERROR_GMSA_IMPORT_CCGEVENTS -ErrorMessage "Failed to import the CCGEvents.man manifest file."

```

Comment:
Should we keep the exist code WINDOWS_CSE_ERROR_GMSA_IMPORT_CCGEVENTS?

Reply:
Good catch. Since we have not release it, I think that we can remove it and update the error code. Later if we have release a new feature and remove some code, I think that we can remove the error code but should not change any code values.